### DISCUSSION TITLE
How to open link in specified container?

### DISCUSSION THREAD

**Discussion by drselump14** [tridactyl/tridactyl#4672] [Q&A]
I've found a command to manage the container, but I can't find one to open a link in the specified container.

Does that kind of command exist?

Any help would be appreciated

**Comment by bovine3dom** [comment]
You'll need to make yourself a custom hint mode. Untested but

`:hint -W tabopen -c [container name]` should work, otherwise

`:composite hint -pipe a href | tabopen -c [container name]`

You can bind to a key in the normal way, see `:help hint` and `:help bind` for more info :)

---

Having reread your post I'm less sure that that was your question. Maybe you are just looking for `:tabopen -c [container] [URL]`?

**Reply by drselump14** [reply]
Hi, thanks for the answer. Both are what I was looking for. Thanks!

**Reply by fiskhest** [reply]
Is there a clever way to composite this to prompt with a list of possible containers to choose from? I.e. press hint shortcut, select link and then the console prompt opens with a list of the configured containers?

**Reply by fiskhest** [reply]
I created a feature request for the above question: https://github.com/tridactyl/tridactyl/issues/4926
